skin/adminhtml/default/default/sendinblue/js/sendinblue.js

```diff
@@ -602,7 +602,31 @@ jQuery.ajax({
 	}
 });
 });
-
+jQuery(".AutomationSubmit").click(function()
+	{
+		var Automation = jQuery("input[name='Automation']:checked").val();
```

Comment:
Variable name should not start capital latter.

Reply:
Done